atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/WriteInfoPartitioner.java

```diff
@@ -66,18 +80,22 @@ private PartitionInfo getPartitionInfo(WriteInfo write) {
 
     @VisibleForTesting
     TableMetadataPersistence.SweepStrategy getStrategy(WriteInfo writeInfo) {
-        return cache.computeIfAbsent(writeInfo.tableRefCell().tableRef(), this::getStrategyFromKvs);
+        return cache.getUnchecked(writeInfo.tableRefCell().tableRef());
     }
 
     private TableMetadataPersistence.SweepStrategy getStrategyFromKvs(TableReference tableRef) {
-        // todo(gmaretic): fail gracefully if we cannot hydrate? How -- return NOTHING?
-        return TableMetadata.BYTES_HYDRATOR.hydrateFromBytes(kvs.getMetadataForTable(tableRef)).getSweepStrategy();
+        try {
+            return TableMetadata.BYTES_HYDRATOR.hydrateFromBytes(kvs.getMetadataForTable(tableRef)).getSweepStrategy();
+        } catch (Throwable th) {
+            log.warn("Failed to obtain sweep strategy for table {}. Assuming sweep strategy is CONSERVATIVE.",
+                    UnsafeArg.of("tableRef", tableRef), th);
```

Comment:
can we `LoggingArgs` the table ref? I guess there's not much point since the metadata is unloadable though